packages/core/src/shipping/shipping-country-request-sender.ts

```diff
@@ -7,10 +7,12 @@ export default class ShippingCountryRequestSender {
     constructor(private _requestSender: RequestSender, private _config: { locale?: string }) {}
 
     loadCountries(
-        channelId: number,
+        channelId: number | null,
```

Comment:
```suggestion
        channelId?: number,
```

Reply:
I believe TS will complain about this in the shipping-action-country-selector.ts about it not being assignable to type number.

Reply:
Didn't check that file.
As you said, this cannot be optional here.